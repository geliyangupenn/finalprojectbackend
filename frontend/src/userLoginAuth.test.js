import React from 'react';
import { render, screen } from '@testing-library/react';

import LoginPage from './userLoginAuth';

const renderer = require('react-test-renderer');

// snapshot testing
test('LoginPage matches snapshot', () => {
  const component = renderer.create(<LoginPage/>);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
