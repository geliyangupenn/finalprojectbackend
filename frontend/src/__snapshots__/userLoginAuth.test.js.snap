// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoginPage matches snapshot 1`] = `
Array [
  <h1
    id="h1"
  >
    User Login
  </h1>,
  <div>
    <label
      for="name"
    >
      UserName: 
    </label>
    <input
      id="name"
      name="name"
      type="text"
    />
  </div>,
  <div>
    <label
      for="password"
    >
      Password: 
    </label>
    <input
      id="password"
      name="password"
      type="password"
    />
  </div>,
  <button
    onClick={[Function]}
    type="submit"
  >
    Login
  </button>,
  <button
    id="resetpassword"
    type="submit"
  >
    forgot password?
  </button>,
  <h2
    id="h2"
  />,
]
`;
